src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -1184,6 +1184,7 @@ public Component getTableCellRendererComponent(JTable table, Object value,
             // TODO: it's rather a temporary trick, to be revised
             String text;
             Object[] objs = new Object[1];
+            nf.setMinimumFractionDigits(1);
```

Comment:
I'd rather put that into constructor. It's to be run once rather than each time `getTableCellRendererComponent` is called.